README.md

```diff
@@ -2,62 +2,111 @@
 
 # End to end tests for Archive.org using [Playwright](https://playwright.dev/)
 
+
 ## (Optional) BrowserStack Local Setup
 
-- login to BrowserStack and retrieve the account username and access key, see documentation [here](https://www.browserstack.com/docs/automate/playwright/getting-started/nodejs/test-runner)
+- Login to BrowserStack and retrieve the account username and access key, see documentation [here](https://www.browserstack.com/docs/automate/playwright/getting-started/nodejs/test-runner)
 
 
-## Running tests locally
+## Local Setup
 
-- install dependencies:
+- Install dependencies:
 
     `npm i`
 
-- install Playwright browser libs:
+- Install Playwright browser libs:
 
     `npx playwright install`
 
-- run all the tests in headless mode and generate 1 whole test report:
+- Create a `.env` file by copying the contents from `.env.sample` and add the respective values you want to use for testing
+
+    - this is required to run tests with loggedIn flows
+
+
+## Running tests
+
+- Run all tests: `npm run test`
+
+- Run tests by category:
+
+    - This will run a custom script that can take different shell arguments/parameters for the following:
+        > Note:
+        >   - The arrangement/order for passing of arguments can be jumbled for flexibility
+        >   - The script is somewhat patterned with the current Playwright commands that uses `--` convention
+        >   - It's important to add `--` convention in CLI to separate the arguments from the `npm script` to be passed to the custom script
+        
+        - Test category (by folder structure):
+            - It will run all tests if you didn't specify the test category you want to test
+            - This can only execute 1 category in 1 command
+            - Refer to the folder names in `tests` directory (`tests/*`), except for `tests/page-objects`
+            - Sample parameters: `about, av, books, collection, details, donation, home, login, music, profile, search`
+            - Sample execution:
+
+                `npm run test about`
+
+                `npm run test books`
 
-    `npm run test`
+                `npm run test donation`
 
-- run all the tests in headless mode by each category and generate test report by category:
+                `npm run test profile`
 
-    `./run-tests.sh`
+        - Test execution mode:
+            - The tests will run in headless mode by __default__ which will execute the tests in terminal only
+            - Headed mode will spawn a browser window in the screen and execute the tests
+            - Accepted parameters: `--headed`
+            - Sample execution:
 
-- run all the tests in headed mode (this will load multiple browsers):
+                `npm run test -- about --headed`
 
-    `npm run test:headed`
+                `npm run test about -- --headed`
```

Comment:
QUESTION: As noted above, it is unclear why "--" is necessary, especially since you show two examples with seemingly contradictory placements, before and after "about".

Reply:
updated the code for consistency